fishtank/src/cluster.ts

```diff
@@ -33,14 +37,38 @@ export class Cluster {
     })
   }
 
-  async spawn(options: { name: string; image?: string }): Promise<Node> {
+  async spawn(options: {
+    name: string
+    image?: string
+    config?: Partial<ConfigOptions>
+  }): Promise<Node> {
     const containerName = this.containerName(options.name)
-    await this.backend.runDetached(options.image ?? DEFAULT_IMAGE, {
+
+    const runOptions: RunOptions = {
       name: containerName,
       networks: [this.networkName()],
       hostname: options.name,
       labels: { [CLUSTER_LABEL]: this.name },
-    })
+    }
+
+    if (options.config) {
+      const args: string[] = []
+
+      const configString = JSON.stringify(options.config)
+
+      const dest = join(tmpdir(), 'fishtank', containerName, '.ironfish')
+      await promises.mkdir(dest, {
```

Comment:
This directory is created but never removed. Is the change to remove this directory coming in a future PR? In any case, I would place everything in a subdirectory containing the cluster name, like this:

```
const dest = join(tmpdir(), 'fishtank', this.name, name, '.ironfish')
```

so that during `teardown()` you can safely remove `join(tmpdir(), 'fishtank', this.name)`, without having to find the specific subdirectories.

Reply:
As discussed before here https://github.com/iron-fish/ironfish-tank/pull/8#discussion_r1329076994, this will be in the next PR